ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```diff
@@ -90,7 +90,7 @@
       tags: [jenkins, adoptopenjdk]
       when: jenkins_secret is defined  # Only run if jenkins_secret is defined
     - role: Thunderbird
-      tags: [Thunderbird, jck, adoptopenjdk]
+      tags: jck
```

Comment:
We don't routinely skip the `jck` tag on our runs AFAIK so I'm not sure this is the right approach.

Reply:
Thunderbird is explicitly installed in https://github.com/temurin-compliance/infrastructure/blob/master/ansible/playbooks/JCK_Playbook/roles/thunderbird/tasks/main.yml so I don't believe it should be needed here in our main adoptium playbooks. @fredg02 Can you confirm please?

Reply:
> We don't routinely skip the `jck` tag on our runs AFAIK so I'm not sure this is the right approach.

what do you think to me adding a variable, so the role is disabled by default, unless you override or change the variable as a solution to this?

Reply:
At the moment I say remove it, as the jck machines install it via another mechanism anyway :-)
But would be good to get confirmation from Fred first

Reply:
Removed, and issue logged with EF infra team for confirmation.. https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3934